src/dynarec/la64/dynarec_la64_helper.h

```diff
@@ -88,6 +88,18 @@
         LDxw(hint, wback, fixedaddress);                                                                                                \
         ed = hint;                                                                                                                      \
     }
+// GETEDW can use hint for wback and ret for ed. wback is 0 if ed is xEAX..xEDI
+#define GETEDW(hint, ret, D)                                                                                                            \
```

Comment:
Is this GETEDW used?

Reply:
yeah, the RCR opcode implemented in this PR

Reply:
Oh, I miised it...